sdk/tests/deqp/functional/gles3/es3fPixelBufferObjectTest.js

```diff
@@ -394,7 +401,7 @@ var tcuImageCompare = framework.common.tcuImageCompare;
 
         gl.bindBuffer(gl.PIXEL_PACK_BUFFER, pixelBuffer);
         gl.bufferData(gl.PIXEL_PACK_BUFFER, readReference.getLevel(0).getDataSize(), gl.STREAM_READ);
-        gl.readPixels(0, 0, width, height, readPixelsFormat, readPixelsType, null);
+        gl.readPixels(0, 0, width, height, readPixelsFormat, readPixelsType, 0);
```

Comment:
The original code will call into this one: 
void readPixels(GLint x, GLint y, GLsizei width, GLsizei height, GLenum format, GLenum type, ArrayBufferView? pixels)
In this API, pixel buffer should not be bound. 

After the revision, it will call into the correct one: 
void readPixels(GLint x, GLint y, GLsizei width, GLsizei height, GLenum format, GLenum type, GLintptr offset);